doc/guide.rst

```diff
@@ -153,6 +156,33 @@ Some of these limitations will be lifted in the future, as time allows.
 .. versionadded:: 1.18
     ``--cores=0`` for autodetection
 
+Speed-up tricks
+---------------
+
+There are several tricks for limiting wall-clock time while using cutadapt.
+
+``-Z`` (shorthand for ``--compression-level 1``) can be used to limit the
+amount of cpu time which is spent on the compression of output files.
+Alternatively, choosing filenames not ending with ``.gz``, ``.bz2`` or ``.xz``
+will make sure no cpu time is spent on compression at all. NOTE: on systems
```

Comment:
cpu -> CPU

Reply:
And remove "NOTE:". There’s no need to state that you’re going to say something. Just say it ;-).